src/analyze.cpp

```diff
@@ -2648,6 +2649,29 @@ static Error resolve_enum_zero_bits(CodeGen *g, ZigType *enum_type) {
                     buf_sprintf("consider 'union(enum)' here"));
         }
 
+        AstNode *tag_value = field_node->data.struct_field.value;
+
+        if (buf_eql_str(type_enum_field->name, "_")) {
+            if (decl_node->data.container_decl.init_arg_expr == nullptr) {
+                add_node_error(g, field_node, buf_sprintf("non-exhaustive enum must specify size"));
+                enum_type->data.enumeration.resolve_status = ResolveStatusInvalid;
+            }
+            if (log2_u64(field_count - 1) == enum_type->size_in_bits) {
```

Comment:
If `field_count` becomes 0 you're going to hit either:
* Undefined behavior with gcc/clang as `log2_u64` calls `clzll` that in turn calls `__builtin_clzll` and that's [undefined](https://gcc.gnu.org/onlinedocs/gcc/Other-Builtins.html) if the parameter is zero
* Garbage with MSVC as `_BitScanReverse` won't write anything in the `lz` variable

I'd say that the `clzll` definition in `util.hpp` needs to be fixed